CMake/ctest_script.cmake

```diff
@@ -74,6 +74,12 @@ ELSEIF( ${HOSTNAME} MATCHES "billmp1" )
     ELSE()
         SET( CTEST_BUILD_NAME Clang-Release-Mac )
     ENDIF()
+ELSEIF( ${HOSTNAME} MATCHES "ryzen-box" )
```

Comment:
OK if it makes development+testing easier. It does raise the question about why these settings are needed, but that could be asked of any of the machine settings in the ctest script...